cmds/rush/tty_linux.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+	"unsafe"
+)
+
+const (
+	sysIoctl = 16
```

Comment:
Rush is still a handy way to test things. Think of it as the equivalent of ash, as compared to bash.